pages/index.js

```diff
@@ -26,9 +26,9 @@ export default function Home() {
         </p>
 
         <div className="call-to-action">
-          <a className="button-primary" href="https://forms.gle/3N3CfqbH6YkM1DF67"> Join us! </a>
+          <a className="button-secondary" href="https://forms.gle/3N3CfqbH6YkM1DF67"> Join us! </a>
```

Comment:
Should have been more clear. Put the `"button-primary"` on the "Join us!", but put the `"button-secondary"` on the "Become a Mentor!"